### ISSUE TITLE
Can i get my info removed

### ISSUE THREAD

**Issue by AAZig** [Plancke/hypixel-php#57]
Can i get my info removed??? i am currently being stalked in the game and having my info used against me

**Comment by mdashlw** [comment]
Hypixel doesn't remove any info from the API on request

**Comment by johnbentcope** [comment]
I'm receiving too many messages from strangers online asking to buy my account. They all say they found my name through your service. Opting out would be a great feature.

**Comment by mdashlw** [comment]
> I'm receiving too many messages from strangers online asking to buy my account. They all say they found my name through your service. Opting out would be a great feature.

No, it won't. Hypixel already provides a way to remove your stats from the API, for staff only. The fact that it's not available even to youtubers suggests that they are not going to make it public.
